mq/inmemory/mq.h

```diff
@@ -124,7 +131,17 @@ class MMQ final {
   // This method will be called from one thread, which is spawned and owned by an instance of MMQ.
   typedef CONSUMER T_CONSUMER;
 
-  // The only constructor requires the refence to the instance of the consumer of entries.
+  // Constructor that uses default `ProcessMessageConsumer`.
```

Comment:
Nit: `Constructor that uses the default` is a tiny bit more English (run through Gene too). :-)